frontend/src/_styles/theme.scss

```diff
@@ -2519,7 +2519,77 @@ input[type='text'] {
 .code-editor-widget {
   border-radius: 0;
   .CodeMirror {
-    border-radius: inherit;
+    border-radius: 0!important;
     margin-top: -1px!important;
   }
+}
+
+.code-hinter-wrapper  .popup-btn {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+}
+
+.code-hinter-wrapper:hover {
+  .popup-btn {
+    display: block!important;
+    z-index: 99999;
+  }
+}
+
+.modal-portal-wrapper {
+  z-index: 10000000;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  position: absolute;
+  left: 50%;
+  top: 40%;
+  transform: translate(-50%, -50%);
+  height: 500px;
+  width: 500px;
+  padding: 0;
+  
+  .modal-body {
+    width: 500px!important;
+    height: 300px!important;
+    padding: 0px!important
+  }
+  
+  .modal-dialog {
+    margin-top: 4%;
+  }
+  
+  .modal-header {
+    padding: 0;
+    font-size: 14px;
+  }
+  
+  
+  .editor-container{
+    padding: 0px;    
+    .CodeMirror {
+      border-radius: 0;
+      margin: 0;
+      width: 100%!important;
+    }
+  }
+
+}
+
+
+.portal-header {
+  display: flex;
+  align-items: center;
+  padding: 0.5rem 0.75rem;
+  color: #656d77;
+  background-color: #ffffffd9;
+  background-clip: padding-box;
+  border-bottom: 1px solid #0000000d;
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
+}
+
+.CodeMirror-hints {
+  z-index: 999990000!important;
```

Comment:
❌ 

instead have like a variable in scss saying `modal-zindex`, `popover-zindex` such that there is track of zindex in the app, or else it becomes really difficult to overlay another element if needed in the future